compiler/record-minmax/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ if(NOT ENABLE_TEST)
   return()
 endif(NOT ENABLE_TEST)
 
+target_compile_options(record-minmax PUBLIC -fsanitize=thread)
+target_link_libraries(record-minmax -fsanitize=thread)
+
```

Comment:
Ah, this will make record-minmax with sanitizer for all other cases. Could you make a separate executable for thread  test? For example,
 
```suggestion
# record-minmax-for-thread-test is used in record-minmax-thread-safety-test
add_executable(record-minmax-for-thread-test ${DRIVER} ${SOURCES})
target_include_directories(record-minmax-for-thread-test PRIVATE include)
target_link_libraries(record-minmax-for-thread-test arser)
...
target_compile_options(record-minmax-for-thread-test PUBLIC -fsanitize=thread)
target_link_libraries(record-minmax-for-thread-test -fsanitize=thread)
```

You may use `record-minmax-for-thread-test` in `record-minmax-thread-safety-test`.

Reply:
I guess this will resolve CI failure.

BTW, it seems that thread sanitizer found an issue in luci-interpereter. I made https://github.com/Samsung/ONE/issues/10188 for that issue.

Reply:
Done